src/kernels/backproject.cl

```diff
@@ -67,6 +67,9 @@ backproject_tex (read_only image2d_t sinogram,
 #ifdef DEVICE_GEFORCE_GTX_TITAN_BLACK
 #pragma unroll 8
 #endif
+#ifdef DEVICE_GEFORCE_GTX_TITAN
+#pragma unroll 14
+#endif
```

Comment:
Alright, here we go. But anyway, who sets the arguments? When I compile the kernel with PyOpenCL nothing is set, that's why I thought the Titan was just not defined or something.